### DISCUSSION TITLE
重启openwrt后挂盘出现错误

### DISCUSSION THREAD

**Discussion by keyboydg** [AlistGo/alist#4087] [Q&A]
我梳理了一下，应该是跟联网有关，我试过sleep 90做延迟启动，可是还是会出现挂盘错误，不知道问题出在哪里了，不知道作者可不可以出一个选项，挂盘错误之后多少秒之后会重新尝试挂盘的选项？

**Comment by anwen-anyi** [comment]
在启动的时候已经重试了重试5次还是3次来着，
有人遇到和你相似的问题
不过你可以延迟1 2分钟后让AList`重启`一次~ 
sleep 90 是多久（）

**Reply by keyboydg** [reply]
sleep 90就是延迟一分半钟，我现在试一下3分钟吧
